pom.xml

```diff
@@ -118,113 +101,64 @@
-                <enabled>false</enabled>
-            </snapshots>
-        </pluginRepository>
-        <pluginRepository>
-            <id>evolvis-snapshot-repository</id>
-            <name>evolvis.org snapshot repository</name>
-            <url>http://maven-repo.evolvis.org/snapshots</url>
-            <snapshots>
-                <enabled>true</enabled>
-            </snapshots>
-            <releases>
-                <enabled>false</enabled>
-            </releases>
-        </pluginRepository>
-    </pluginRepositories>
-
-    <!-- Repositories -->
-    <repositories>
-        <repository>
-            <id>evolvis-release-repository</id>
-            <name>evolvis.org release repository</name>
-            <url>http://maven-repo.evolvis.org/releases</url>
-        </repository>
-
-        <repository>
-            <id>evolvis-snapshot-repository</id>
-            <name>evolvis.org snapshot repository</name>
-            <url>http://maven-repo.evolvis.org/snapshots</url>
-            <snapshots>
-                <enabled>true</enabled>
-            </snapshots>
-        </repository>
-    </repositories>
-
-    <!--
-        BIG FAT WARNING: Maven IGNORES the user setting herein. You MUST
-        provide a proper setting in .m2/settings.xml instead.
-    -->
     <distributionManagement>
-
         <repository>
-            <id>evolvis-release-repository</id>
-            <name>evolvis.org release repository</name>
-            <url>scpexe://maven@maven-repo.evolvis.org/var/www/maven_repo/releases</url>
+            <id>tarent-maven-releases</id>
+            <url>https://repo-bn-01.lan.tarent.de/repository/maven-releases/</url>
         </repository>
 
         <snapshotRepository>
-            <id>evolvis-snapshot-repository</id>
-            <name>evolvis.org snapshot repository</name>
-            <url>scpexe://maven@maven-repo.evolvis.org/var/www/maven_repo/snapshots</url>
+            <id>tarent-maven-snapshots</id>
+            <url>https://repo-bn-01.lan.tarent.de/repository/maven-snapshots/</url>
         </snapshotRepository>
-
     </distributionManagement>
 
     <dependencies>
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.11</version>
+            <version>4.12</version>
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-compat</artifactId>
+            <version>${maven.version}</version>
+        </dependency>
+
         <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-plugin-api</artifactId>
             <version>${maven.version}</version>
-            <scope>compile</scope>
         </dependency>
 
         <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-project</artifactId>
-            <version>${maven.version}</version>
```

Comment:
Why is the version not needed here anymore?

Reply:
ups, actually I set it to 2.0.11 but we do not need the whole dependency anymore. I will remove it.

Reply:
done